common/src/main/java/io/pravega/common/Exceptions.java

```diff
@@ -212,6 +212,18 @@ public static void checkNotClosed(boolean closed, Object targetObject) throws Ob
         }
     }
 
+    /**
+     * Logs the exception and its cause to error log.
+     *
+     * @param t   The exception to be logged.
+     */
+    public static void unwrapAndLogException(Throwable t) {
```

Comment:
Why is this method needed? If a throwable has a cause set the cause will automatically be logged when the exception is logged. That log.error line below will log the full chain of causes.

Reply:
There was an issue with the call to `log.error`. Fixed it. We do not need this method. `log.error` fixes this.